### DISCUSSION TITLE
Making a fleet chatty?

### DISCUSSION THREAD

**Discussion by adegans** [endless-sky/endless-sky#7867] [Balance, Mechanics, and Interactions]
Often when flying around you see Navy ships "chat" something in the lower left part where all events and messages scroll by.
Other random ships will shout out things too in chat sometimes.

Is that something a plugin can hook into?
If so, how? I seemingly can't define any chatter for a custom fleet.
Not sure if I should... But it'll be an experiment.

**Comment by quyykk** [comment]
Yeah, that's called a hail and it is defined as a phrase inside the government of the ship. https://github.com/endless-sky/endless-sky/wiki/CreatingGovernments See the various hail attributes.
